staging/src/k8s.io/client-go/tools/clientcmd/loader.go

```diff
@@ -154,9 +158,10 @@ func NewDefaultClientConfigLoadingRules() *ClientConfigLoadingRules {
 	}
 
 	return &ClientConfigLoadingRules{
-		Precedence:       chain,
-		MigrationRules:   currentMigrationRules(),
-		WarnIfAllMissing: warnIfAllMissing,
+		Precedence:               chain,
+		MigrationRules:           currentMigrationRules(),
+		WarnIfAllMissing:         warnIfAllMissing,
+		WarnIfAllMissingCallback: klog.V(4).Infof,
```

Comment:
previously it was klog.Warningf, why switching to klog.V(4).Infof?

Reply:
I did not want to commit (I was just checking that `klog.V(x).Infof` works as well and I committed that line with no purpose). Restored to `klog.Warningf`.